lib/rubygems/request.rb

```diff
@@ -40,7 +40,7 @@ def proxy_uri; @connection_pool.proxy_uri; end
   def cert_files; @connection_pool.cert_files; end
 
   def self.get_cert_files
-    pattern = File.expand_path("./ssl_certs/*.pem", File.dirname(__FILE__))
+    pattern = File.expand_path("./ssl_certs/**/*.pem", File.dirname(__FILE__))
```

Comment:
Turns out `ssl_certs/*/*.pem` works fine here, since we aren't nesting directories. I updated the Bundler PR to only use one star, too.